README.md

```diff
@@ -1,42 +1,90 @@
 <img src="https://cdn.rawgit.com/akveo/react-native-ui-kitten/085afb52/docs/assets/banner.png"/>
 
-# react-native-ui-kitten [![Build Status][badge-travis]][link-travis] [![Coverage Status][badge-coveralls]][link-coveralls] [![Dependency Status][badge-david]][link-david] [![npm][badge-downloads]][link-downloads] [![npm][badge-license]]()
+# UI Kitten [<img src="https://i.imgur.com/oMcxwZ0.png" alt="Eva Design System" height="20px" />][link:eva] [![npm][badge:license]]() [![Build Status][badge:travis]][link:travis] [![Coverage Status][badge:coveralls]][link:coveralls]
 
-This branch contains sources of something really awesome.
-We do what we did before but in a new way.
+[Documentation][link:doc-homepage]
 
-## What's new will you have:
-- TypeScript support
-- Full test coverage of the framework
-- Better code quality
-- Better development environment for contributors
-- Better theme system
+UI Kitten is a React Native framework for creating stunning cross-platform mobile applications.
+This UI Library based on Eva Design System which brings consistency and scalability in the design and development process. 
+It contains a set of general purpose UI components styled in a similar way. 
+And the most awesome thing: the themes can be changed in the runtime, without any need of reloading the application.
 
-## How can I support the developers?
-- Star our GitHub repo
-- Create pull requests, submit bugs, suggest new features or documentation updates
-- Follow us on [Twitter](https://twitter.com/akveo_inc)
-- Like our page on [Facebook](https://www.facebook.com/akveo/)
+100% Free and Open Source!
+
+## What's included
+
+- **Almost 20 general purpose components** designed and tested to save you time while. 
+
+- **Comprehensive clear documentation** with the tons of examples.
+
+- **Theming System**. Use Light and modern Dark themes and create your own ones.
+
+- **Eva Design System Support**. Construct any interface using basic components following Eva Guidelines and it will always have a stunning design. 
```

Comment:
- **Eva Design System Support**. Construct an interface using basic components following Eva specifications and it will always have a stunning design.